tests/lib/prepare.sh

```diff
@@ -608,12 +608,14 @@ prepare_ubuntu_core() {
     fi
 
     # Wait for the snap command to become available.
-    for i in $(seq 120); do
-        if [ "$(command -v snap)" = "/usr/bin/snap" ] && snap version | grep -q 'snapd +1337.*'; then
-            break
-        fi
-        sleep 1
-    done
+    if not [ "$SPREAD_BACKEND" = "external" ]; then
```

Comment:
Using `if [ "$SPREAD_BACKEND" != "external" ]; then` would also work here and is (to me) slightly more canonical but its a nitpick :)

Reply:
Yeah, `[ "$SPREAD_BACKEND" != "external" ]` is the right way to do it.

Reply:
Pushed a little patch to address this.